content/wasm-functions/using-cron-jobs.md

```diff
@@ -0,0 +1,237 @@
+title = "Scheduling Tasks with Cron Jobs in Spin"
+template = "functions_main"
+date = "2025-03-25T00:00:01Z"
+enable_shortcodes = true
+
+---
+- [Prerequisites](#prerequisites)
+- [Introduction to Cron Jobs](#introduction-to-cron-jobs)
+  - [What Are Cron Jobs?](#what-are-cron-jobs)
+- [Create a Spin Application](#create-a-spin-application)
+  - [Deploy Spin Application to Fermyon Wasm Functions](#deploy-spin-application-to-fermyon-wasm-functions)
+- [Scheduling a Cron Job for Your Spin Application](#scheduling-a-cron-job-for-your-spin-application)
+- [Managing Cron Jobs](#managing-cron-jobs)
+- [Delete Cron Job](#delete-cron-job)
+- [Next Steps](#next-steps)
+  
+This tutorial guides you through the process of scheduling HTTP requests in a Spin application running on Fermyon Wasm Functions using cron jobs. The `spin aka crons` command allows applications to receive scheduled HTTP triggers without relying on external services. In this tutorial, you'll learn how to define and manage cron jobs using the `spin aka crons` command.
+
+# Prerequisites
+Before proceeding, ensure you have the following:
+
+- Spin and the `spin aka` plugin installed on your machine ([Installation Guide](./quickstart.md))
+- Access to Fermyon Wasm Functions preview ([Sign Up Form](https://fibsu0jcu2g.typeform.com/fwf-preview))
+
+# Introduction to Cron Jobs
+
+## What Are Cron Jobs?
+
+Cron jobs are scheduled tasks that run at specified intervals. They are commonly used for automating repetitive tasks such as data processing, report generation, and background maintenance. Other potential use cases include: 
+
+* Automated data backups
+* Periodic API polling
+* Log file rotation
+* Scheduled notifications
+
+# Create a Spin Application 
+
+You must have a Spin application deployed to Fermyon Wasm Functions to associate a cron job. We'll create a new Spin application using the `http-js` template, which we will then invoke on a schedule using a cron job in later steps:
+
+<!-- @selectiveCpy -->
+
+```console
+$ spin new -t http-js --accept-defaults hello-cron-job
+
+$ cd hello-cron-job
+
+$ npm install
+```
+
+We'll go ahead add a path that will be used our cron job in later steps:
+
+```toml
```

Comment:
Do we need to add a second trigger?  Is this trying to illustrate a recommended practice?  It would be good to say more about the motivation because right now this feels like a step I follow without knowing why.

Reply:
I went back and forth on this... I decided to not add a second trigger to keep the example simple. Perhaps I can write a more realistic example for Spin Hub that we can link as a resource at the end of the tutorial for those interested in that scenario. 


To simplify things, I've removed the specific path and replaced it with wildcard.